annotations/templates/annotations/project_list.html

```diff
@@ -5,6 +5,38 @@
 <script src="{% static "annotations/js/spin.min.js" %}"></script>
 <script src="{% static "annotations/js/jquery.spin.js" %}"></script>
 <script src="{% static "annotations/js/cytoscape.min.js" %}"></script>
+
+<script>
+$(document).ready(function() {
+    // Handle clicks on project links for text import
+    $('.project-import-link').click(function(e) {
+        e.preventDefault();
+        var url = $(this).attr('href');
+        
+        // loading spinner
+        $(this).spin();
+        
+        // Make AJAX call to import text
+        $.ajax({
+            url: url,
+            method: 'GET',
+            success: function(response) {
+                if (response.success && response.redirect_url) {
+                    // Redirect to annotation page
+                    window.location.href = response.redirect_url;
+                }
+            },
+            error: function(xhr, status, error) {
+                console.error('Error importing text:', error);
+                alert('Failed to import text. Redirecting to projects page...');
```

Comment:
javascript alerts are really ugly. if the view redirects appropriately after text import, then the error message can be shown on the next screen easily.